django/contrib/admin/utils.py

```diff
@@ -490,10 +490,20 @@ def construct_change_message(form, formsets, add):
     Translation happens later on LogEntry access.
     """
     change_message = []
```

Comment:
I think I would put `change_message = []` below the new code so it's not separated from where it's used.